apps/vaporgui/RenderHolder.cpp

```diff
@@ -212,6 +212,8 @@ RenderHolder::RenderHolder(QWidget *parent, ControlExec *ce, const vector<QWidge
     _vaporTable = new VaporTable(tableWidget, false, true);
     _vaporTable->Reinit((VaporTable::ValidatorFlags)(0), (VaporTable::MutabilityFlags)(VaporTable::IMMUTABLE), (VaporTable::HighlightFlags)(VaporTable::ROWS));
     _vaporTable->ShowToolTips(true);
+    _vaporTable->StretchToColumn(2);
+    _vaporTable->HideColumn(1);
     _currentRow = 0;
 
```

Comment:
```suggestion
tableWidget->setWordWrap(false);```

This will resolve the word wrapping with the LeighOrf 24MayControlRun data set

Reply:
Thanks John.  Fixed.